config/application.rb

```diff
@@ -42,5 +42,19 @@ class Application < Rails::Application
 
     # YAML Defaults for CSV
     config.csv_defaults = YAML.load_file(Rails.root.join('config', 'csv_file_defaults.yml'))
+
+    # React Settings
+    # Settings for the pool of renderers:
+    config.react.server_renderer_pool_size  ||= 1  # ExecJS doesn't allow more than one on MRI
+    config.react.server_renderer_timeout    ||= 20 # seconds
+    config.react.server_renderer = React::ServerRendering::BundleRenderer
+    config.react.server_renderer_options = {
+        files: ["server_rendering.js"],       # files to load for prerendering
+        replay_console: true,                 # if true, console.* will be replayed client-side
+    }
+    # Changing files matching these dirs/exts will cause the server renderer to reload:
+    config.react.server_renderer_extensions = ["jsx", "js"]
+    config.react.server_renderer_directories = ["/app/assets/javascripts", "/app/javascript/"]
+    config.react.camelize_props = true # default false
```

Comment:
Does this block of configuration mean that Rails is going to serve up the React bits as well? No additional nodejs server required?

Reply:
These are the default configurations values when using https://github.com/reactjs/react-rails#server-side-rendering

I added them to here even though they are defaults for awareness of configuration that can change